spark-plugins/src/test/java/co/cask/hydrator/plugin/spark/test/SparkPluginTest.java

```diff
@@ -124,7 +126,8 @@ public static void setupTest() throws Exception {
     addPluginArtifact(NamespaceId.DEFAULT.artifact("spark-plugins", "1.0.0"), parents,
                       NaiveBayesTrainer.class, NaiveBayesClassifier.class,
                       KafkaStreamingSource.class, KafkaUtils.class, DefaultDecoder.class,
-                      TwitterStreamingSource.class);
+                      TwitterStreamingSource.class, InitialPositionInStream.class,
```

Comment:
How is the config test related to the source sink being added here ? 
There should be a unit test for the plugin ? Is it not possible to  use kinesis streaming source/sink in memory by starting a in memory kinesis stream or something ?